compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -18,20 +17,8 @@ include(${KERNEL_REGISTER_FILE})
 add_library(${LUCI_INTERPRETER_KERNELS} STATIC ${SOURCES})
 set_target_properties(${LUCI_INTERPRETER_KERNELS} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 target_include_directories(${LUCI_INTERPRETER_KERNELS} PUBLIC ${LUCI_INTERPRETER_SOURCE_DIR})
-target_include_directories(${LUCI_INTERPRETER_KERNELS} SYSTEM PRIVATE
-        "${TensorFlowRuySource_DIR}"
-        "${TensorFlowGEMMLowpSource_DIR}"
-        "${TensorFlowEigenSource_DIR}"
-        "${TensorFlowSource_DIR}")
-
-if (NOT LUCI_MICRO)
-  target_link_libraries(${LUCI_INTERPRETER_KERNELS}
-          PUBLIC ${LUCI_INTERPRETER_CORE}
-          PRIVATE nncc_common Threads::Threads)
```

Comment:
As I understand, linkage with Threads already done in *linux/pal.cmake*, but what happens with nncc_common?

Reply:
hmmmm, I missed this one.
This library does not provide any code, just some flags. I'll check if luci-micro builds with it.

Reply:
looks like arm build works with nncc too!
bringing it back

Reply:
Thanks) 
Please, note that I'm asking about `nncc_common` missing in  linux configuration. I think it's ok to reduce linkage size for mcu pal.

Reply:
too late, I enabled it everywhere...
let's disable it if needed later =)